src/commands/config/ConfigAfkCheck.ts

```diff
@@ -62,8 +62,6 @@ export class ConfigAfkCheck extends BaseCommand {
 
     /** @inheritDoc */
     public async run(ctx: ICommandContext): Promise<number> {
-        if (!(ctx.channel instanceof TextChannel)) return -1;
```

Comment:
is this necessary?  I thought we can enforce voice channel selection from the arguments.